pkg/minikube/driver/endpoint.go

```diff
@@ -30,6 +32,10 @@ func ControlPlaneEndpoint(cc *config.ClusterConfig, cp *config.Node, driverName
 		port, err := oci.ForwardedPort(cc.Driver, cc.Name, cp.Port)
 		hostname := oci.DefaultBindIPV4
 		ip := net.ParseIP(hostname)
+		if ip == nil {
+			errMsg := fmt.Sprintf("failed to parse ip")
```

Comment:
since the var errMsg is not used anywhere else, consider just putting it in the Wrap